RPi3-installation.md

```diff
@@ -0,0 +1,28 @@
+# Installing LibreElec, KODI, and Netflix on a Raspberry Pi
+
+These instructions assume you have an unused Raspberry Pi and are starting from scratch. The steps will wipe whatever is on the Raspberry Pi's SD card.
+
+This has been tested on a Raspberry Pi 3, but the steps are likely to be very similar if not the same for other models.
+
+## Installation the latest Alpha LibreElec and KODI 18
+
+* Download the [LibreElec USB-SD Creator](https://libreelec.tv/downloads/)
+* Install the latest LibreElec stable version onto your Raspberry Pi's SD card using the USB-SD Creator 
+* Insert the SD card into the Raspberry Pi and boot it
+* Enable SSH and Samba (SMB) options when asked - make note of the default user and password
+* Download the [latest LibreElec Alpha](https://forum.kodi.tv/showthread.php?tid=298461) (These instructions were tested using 0722, but the latest build should generally work)
```

Comment:
You might want to add "LibreELEC Settings add-on Development Updates
For these builds since #0612, enter a URL into a Custom Channel, selecting from the following URLs based on your hardware:
Code:
RPi Zero/RPi1: http://milhouse.libreelec.tv/builds/master/RPi
RPi2/RPi3:     http://milhouse.libreelec.tv/builds/master/RPi2
Generic (x86): http://milhouse.libreelec.tv/builds/master/Generic
then select "Milhouse-9.0" as the Update Channel (or "Milhouse-8.0" for builds before #0202)." from the forum for the automatic updates.